parlai/core/torch_ranker_agent.py

```diff
@@ -395,6 +395,9 @@ def _build_candidates(self, batch, source, mode):
             cands = self.fixed_candidates
             cand_vecs = self.fixed_candidate_vecs
 
+            if self.opt.get('encode_candidate_vecs', False):
```

Comment:
this is not quite right and introduces a bug. cand_vecs are not the model encoded candidates, they are simply the tokenized candidates